### PR TITLE
change http to https for mass.local

### PR DESCRIPTION
**Pull request by clairesunstudio**
Fixed:
  - description: Update http://mass.local to https://mass.local.
    issue: DP-23002

### CODE REVIEW

**Comment by clairesunstudio** [comment]
`Puppeteer encountered an error while running scenario "Document"
Error: net::ERR_CONNECTION_REFUSED at https://mass.local/media/1268726?cachebuster=4l9mb` puppeteer is not able to access https://mass.local

File: .circleci/nightcrawler/nightcrawler.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ switch(target) {
     alias = '@prod';
     break;
   case 'local':
-    base = 'http://mass.local';
+    base = 'https://mass.local';
```
Comment (weitzman) [inline .circleci/nightcrawler/nightcrawler.js]: I think http://mass-web is correct here

File: .ahoy.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ commands:
 
   blackfire:
     cmd: ahoy -f ${MASS_DEV_ENV:=docker}.ahoy.yml blackfire "$@"
-    usage: "Profile CLI requests. Usages: blackfire curl http://mass.local OR blackfire run --samples=10 drush st"
+    usage: "Profile CLI requests. Usages: blackfire curl https://mass.local OR blackfire run --samples=10 drush st"
```
Comment (weitzman) [inline .ahoy.yml]: I think http://mass-web is correct here

Comment (clairesunstudio) [reply .ahoy.yml]:
```suggestion
    usage: "Profile CLI requests. Usages: blackfire curl http://mass-web OR blackfire run --samples=10 drush st"
```

File: .circleci/nightcrawler/nightcrawler.js (hunk 1 above)
Comment (clairesunstudio) [reply .circleci/nightcrawler/nightcrawler.js]:
```suggestion
    base = 'http://mass-web';
```

**Pull request merged**